### PR TITLE
Added grey subheadings in side navigation

### PR DESCRIPTION
**Pull request by karypun**
In the patterns and accessibility sections, grey subheadings have been added to separate different areas of information

### CODE REVIEW

File: apps/docs/src/common/pages/accessibility/resources.tsx [hunk 1]
```diff
@@ -18,20 +18,35 @@ const Page: FC<PageProps> = ({ location }) => (
...
-          href: '/accessibility/audio-and-video',
-          text: 'Audio and video'
-        },
         {
           href: '/accessibility/links',
           text: 'Links'
         },
         {
-          href: '/accessibility/resources',
-          text: 'Resources'
```
Comment (daniel-ac-martin) [inline apps/docs/src/common/pages/accessibility/resources.tsx:32]: Should this `resources.tsx` file be deleted?

File: apps/docs/src/common/pages/accessibility/tables.tsx [hunk 2]
```diff
@@ -0,0 +1,175 @@
+import { FC, createElement as h } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { PageProps } from '@not-govuk/app-composer';
+import { A, NavigationMenu } from '@not-govuk/components';
+
+const tables = require('../../../../assets/images/accessibility/tables.png').default;
+
+export const title = 'Links';
+const description = 'Accessibility guidance for links content';
```
Comment (daniel-ac-martin) [inline apps/docs/src/common/pages/accessibility/tables.tsx:9]: Are these correct? - They don't seem to pertain to tables.

File: apps/docs/src/common/pages/accessibility/resources.tsx (hunk 1 above)
Comment (daniel-ac-martin) [reply apps/docs/src/common/pages/accessibility/resources.tsx:32]: Oh sorry, it's linked to elsewhere. Ignore me.

File: apps/docs/src/common/pages/accessibility/tables.tsx (hunk 2 above)
Comment (karypun) [reply apps/docs/src/common/pages/accessibility/tables.tsx]: I've updated this along with the other page titles and descriptions where needed

**Review by daniel-ac-martin — APPROVED** [review]

**Pull request merged**